boards/arm64/xenvm/xenvm_defconfig

```diff
@@ -33,9 +33,15 @@ CONFIG_MAIN_STACK_SIZE=32768
 CONFIG_IDLE_STACK_SIZE=32768
 CONFIG_ISR_STACK_SIZE=32768
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=32768
-CONFIG_CMSIS_THREAD_MAX_STACK_SIZE=32768
-CONFIG_CMSIS_V2_THREAD_MAX_STACK_SIZE=32768
-CONFIG_CMSIS_V2_THREAD_DYNAMIC_STACK_SIZE=32768
-CONFIG_IPM_CONSOLE_STACK_SIZE=32768
```

Comment:
Looks like spurious change. If you really need to remove all those lines - put them into separate commit.